src/ui/components/Icon/styles.scss

```diff
@@ -80,6 +80,9 @@
 
 .Icon-anonymous-user {
   @include icon($name: "anonymous-user");
+
+  height: 50px;
```

Comment:
This is the right fix but I wonder if it's the wrong place to put it. This will override this icon size everywhere.

A default size that's larger for this icon seems fine, but I think there's already a style in the UserProfile styles.

The mocks say it should be `64px` though. If you change it to `64px` that'd be great, and in `UserProfile/styles.scss` instead of here.

Reply:
UserProfile picture container is already 64x64:
<img width="371" alt="screen shot 2018-03-28 at 12 55 45 pm" src="https://user-images.githubusercontent.com/31850821/38044367-87d0d970-3287-11e8-963b-f00bd7ab43c1.png">

It seems like icon-anonymous user is the cause of this issue
<img width="319" alt="screen shot 2018-03-28 at 12 56 15 pm" src="https://user-images.githubusercontent.com/31850821/38044376-8e9f100a-3287-11e8-8f80-1315a42ab4e8.png">

Reply:
Yes, but you should target the icon in the UserProfile SCSS. I would recommend adding a `className` to the `<Icon />` in `UserProfile` and selecting it that way. We only want this change for the UserProfile, not site-wide.

Reply:
thanks for your clarifications, misunderstood it a bit and sorry for taking your issue 😅 Next time I will check if anyone is assigned